SoftLayer/managers/network.py

```diff
@@ -171,7 +171,10 @@ def add_subnet(self, subnet_type, quantity=None, endpoint_id=None, version=4,
                     item.get('capacity') == quantity_str,
                     version == 4 or (version == 6 and
                                      desc in item['description'])]):
-                price_id = item['prices'][0]['id']
+                if version == 4 and subnet_type == 'static':
+                    price_id = item['prices'][1]['id']
```

Comment:
How do you know this is the right pice id?


From the API I see something like this:
```
   {
        "capacity": "8",
        "description": "8 Static Public IP Addresses",
        "hardwareGenericComponentId": null,
        "id": 583,
        "itemCategory": {
            "categoryCode": "static_sec_ip_addresses",
            "id": 53,
            "name": "Public Secondary Static IP Addresses",
            "quantityLimit": 0,
            "sortOrder": 35
        },
        "itemTaxCategoryId": 166,
        "keyName": "8_STATIC_PUBLIC_IP_ADDRESSES",
        "longDescription": null,
        "prices": [
            {
                "currentPriceFlag": null,
                "hourlyRecurringFee": null,
                "id": 17242,
                "itemId": 583,
                "laborFee": "0",
                "locationGroupId": null,
                "onSaleFlag": null,
                "oneTimeFee": "0",
                "quantity": null,
                "recurringFee": "8",
                "setupFee": "0",
                "sort": 0,
                "termLength": null,
                "tierMinimumThreshold": null
            },
            {
                "currentPriceFlag": null,
                "hourlyRecurringFee": null,
                "id": 1090,
                "itemId": 583,
                "laborFee": "0",
                "locationGroupId": null,
                "onSaleFlag": null,
                "oneTimeFee": "0",
                "quantity": null,
                "recurringFee": "8",
                "setupFee": "0",
                "sort": 0,
                "termLength": null,
                "tierMinimumThreshold": null
            },
            {
                "currentPriceFlag": null,
                "hourlyRecurringFee": ".01",
                "id": 2020,
                "itemId": 583,
                "laborFee": "0",
                "locationGroupId": null,
                "onSaleFlag": null,
                "oneTimeFee": "0",
                "quantity": null,
                "recurringFee": null,
                "setupFee": "0",
                "sort": 0,
                "termLength": null,
                "tierMinimumThreshold": null
            }
        ],
        "softwareDescriptionId": null,
        "units": null,
        "upgradeItemId": null
```

The last one looks to be hourly pricing (didn't know we did hourly subnets....), but I'd like to have a better way of determining were picking the right price. I don't trust the API to always return this price data in the same order is my main concern.

Reply:
Done.